### PR TITLE
(maint) docs update for backward incompatibility change for node parameter

### PR DESCRIPTION
**Pull request by sheenaajay**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -182,6 +182,7 @@ plan myapp::patch (
   })
 }
 ```
+*Note.* In Bolt `--nodes` parameter is deprecated in favour of `--targets`. It is not compatible if you explicitly pass the parameter `nodes`. Please be aware that if you are calling the plan using the positional target arguments (`-n` and/or `-t`) it will still function as intended.
```
Comment (donoghuc) [inline README.md]: The last sentence does not make sense. `-n` is a command line argument (which is deprecated as well as `--nodes`). The positional target arguments describe plan action invocation signatures for example `run_task('my::task', $targets, {'my_arg' => 1})`.

**Comment by lucywyman** [comment]
@sheenaajay Thank you for putting this up! I updated #235 with a note in the README about versioning - I think we can close this in favor of that? I think updating the README for the release is considered part of release prep, it doesn't seem out of place.

**Comment by sheenaajay** [comment]
Thank you @lucywyman

**Comment by sheenaajay** [comment]
Closing this PR . Thanks @lucywyman and @donoghuc for the comments. Since we have docs updated as part of release prep https://github.com/puppetlabs/puppetlabs-reboot/pull/235

**Pull request closed without merging**